torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -3151,8 +3160,9 @@ at::Tensor XLANativeFunctions::upsample_bilinear2d_backward(
   std::vector<int64_t> scaled_output_size =
       torch::lazy::ToVector<int64_t>(output_size);
   if ((scales_h && *scales_h != 1.0) || (scales_w && *scales_w != 1.0)) {
-    scaled_output_size = GetOutputSizeWithScale(input_size, scales_h, scales_w,
-                                                scaled_output_size);
+    scaled_output_size = GetOutputSizeWithScale(
+        input_size, scales_h, scales_w, scaled_output_size,
+        /*round_with_scale_factor=*/false);
```

Comment:
I think there will be a similar problem with backward pass as in the forward.

Reply:
Thanks, updated!